src/System.Reflection/tests/TestExe/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace System.Reflection.Tests
+{
+    public class TestExeException : Exception
+    {
+    }
+
+    public static class TestExe
+    {
+        private static void Main()
+        {
+            throw new TestExeException();
+        }
```

Comment:
Ok, sounds good.  If you wanted to test the exception behavior as well, you could change the signature to take a string[] args, conditionally throw or return based on some input, and then have multiple test cases with the same exe, passing in different inputs.  But it's fine as is.